sdk/nodejs/src/index.ts

```diff
@@ -6,7 +6,7 @@ export { DVCClient, DVCCloudClient }
 export * from './types'
 
 type DVCOptionsCloudEnabled = DVCOptions & { enableCloudBucketing: true }
-type DVCOptionsLocalEnabled = DVCOptions & { enableCloudBucketing: false }
+type DVCOptionsLocalEnabled = DVCOptions & { enableCloudBucketing?: false }
```

Comment:
did you add the default value somewhere?

Reply:
this will set the default when the value is not defined, verified against the example app